README.md

```diff
@@ -86,6 +79,10 @@ This is a project to learn how to install a front end with React and a backend w
 
 - [Not apply]
 
+## FRONT END URL
+
+- FRONT END URL [@MarcoDDM](https://github.com/MarcoDDM/react-two-apps/pull/1)
```

Comment:
- [ ] Hello, kindly makes sure to add license and customize your readme file for this particular project

![image](https://github.com/MarcoDDM/hello-two-apps/assets/112261237/78b8293b-5456-4994-8ca8-d70986deac8a)